cloud/hack/verify-codegen.sh

```diff
@@ -19,10 +19,11 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 
-DIFFROOT="${SCRIPT_ROOT}/pkg"
-TMP_DIFFROOT="${SCRIPT_ROOT}/_tmp/pkg"
-_tmp="${SCRIPT_ROOT}/_tmp"
+DIFFROOT="${KUBEEDGE_ROOT}/pkg"
+TMP_DIFFROOT="${KUBEEDGE_ROOT}/_tmp/pkg"
+_tmp="${KUBEEDGE_ROOT}/_tmp"
```

Comment:
The scripts can also be moved to root hack?

Reply:
yes, i guess that all the contents behind dir `cloud/hack` need moving to the root hack. And IMHO, it's better to hold those in one separate dir, and which dir name is better? codegen?

Reply:
The `cloud/hack/test-integration.sh` seems can be deleted. `cloud/hack/boilerplate` can move to hack, add add a dir `tools` to place the tool.go, other scripts can place under hack directly.

Reply:
fixed